File: content/1-basics/3-loops/go/main.go
package main

import "fmt"

func main() {
    // For loop
    for i := 0; i < 5; i++ {
        fmt.Println(i)
    }

    // Foreach-like
    fruits := []string{"apple", "banana", "orange"}
    for _, fruit := range fruits {
        fmt.Println(fruit)
    }

    // While-like
    j := 0
    for j < 5 {
        fmt.Println(j)
        j++
    }
}

File: content/1-basics/3-loops/java/Main.java
public class Main {
    public static void main(String[] args) {
        // For loop
        for (int i = 0; i < 5; i++) {
            System.out.println(i);
        }

        // Foreach
        String[] fruits = {"apple", "banana", "orange"};
        for (String fruit : fruits) {
            System.out.println(fruit);
        }
        
        // While
        int j = 0;
        while (j < 5) {
            System.out.println(j++);
        }
    }
}
